MANIFESTO.md

```diff
@@ -11,7 +11,7 @@ The answer to this must be the collectivization of operational knowledge, just a
 From time to time, we can get mentally stuck and may need a sparring partner to think outside the box. Open Operations builds a community of practice to keep the barrier to entry low and create a thriving environment for comfortable exchange. Regardless of individual knowledge level, everyone contributes and enriches this community with their knowledge and individual views. Communities of practice connect through various channels serving both asynchronous and synchronous communication needs in the form of pair operating or on a larger scale to solve these challenges together. Therefore, we are building a network by encouraging organizations to foster and connect internal and external communities of practice.
 
 ## We share knowledge
-[...]
+The availability of knowledge and skilled engineers is the limiting factor for many organizations to adopt, leverage, and successfully operate complex technology. For true self-determination in the ever-growing digital domain, we urgently need to build skills and share the gained knowledge freely and in unlimited ways also in the realm of operational knowledge — exactly the way we started with software code almost half a decade ago. We encourage organizations to collectivize operational knowledge and share it with the entire Open Operations community.
```

Comment:
Last minor nit - this struck me a week ago, however I did not adhere to it, because I was not sure wether it is important or was just the moment. Now re-reading the paragraph I'm stumbling across it again, so I at least want to raise it, however this is by far no blocker:

"For true self-determination in the ever-growing digital domain, we urgently need to build skills[...]"

I would rephrase that and leave the urgently out there:

"For true self-determination in the ever-growing digital domain, we need to build skills[...]"